packages/agent/tsconfig.json

```diff
@@ -21,7 +21,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "moduleResolution": "node",
-    "outDir": "lib"
+    "outDir": "lib/js",
+    "declarationDir": "lib/typings"
```

Comment:
transpile to `lib/js`, generate definition to `lib/typings`. 
`lib/` dir publishes to npm, `lib/build` dir put to CDN.